test/sharness/t0060-daemon.sh

```diff
@@ -121,14 +121,30 @@ test_expect_success "daemon with pipe eventually becomes live" '
   test_fsh cat stdin_daemon_out || test_fsh cat stdin_daemon_err || test_fsh cat stdin_poll_apiout || test_fsh cat stdin_poll_apierr
 '
 
-ulimit -n 512
+ulimit -S -n 512
 TEST_ULIMIT_PRESET=1
 test_launch_ipfs_daemon
 
 test_expect_success "daemon raised its fd limit" '
-	grep "ulimit" actual_daemon > /dev/null
+	grep "raised file descriptor limit to 1024." actual_daemon > /dev/null
 '
 
+get_col_four() {
+	awk '{ print $4 }' $1
+}
+
+if [ `uname` == "Linux" ]; then
```

Comment:
I'm using `syscall.Setrlimit` in https://github.com/jbenet/hang-fds/blob/master/hang-fds.go#L32-L63

```
> ulimit -Sn 256
> ulimit -Sn
256
> ./hang-fds 2048 /ip4/127.0.0.1/tcp/1234
hanging 2048 fds at /ip4/127.0.0.1/tcp/1234
error: failed to raise fd limit to 2058 (still 256)
```